po/fr/gmajctl.po

```diff
@@ -531,22 +531,24 @@ msgid ""
 "General overall mode that determines how fast the Noise Gate should "
 "attenuate/dampen the signal when below Threshold."
 msgstr ""
+"Mode général qui détermine la vitesse à laquelle le Noise Gate devrait"
```

Comment:
il faut que tu fasses attention @jujudusud à conserver les espacements comme dans les chaînes de caractères d'origine.

Par exemple ici : il y a un espace après le "should", et il n'est pas là pour rien. Si ce code était compilé tel quel, et le logiciel utilisé en français, tu aurais phrase qui resssemblerait à ça qui s'afficherait :

> Mode général qui détermine la vitesse à laquelle le Noise Gate devraitatténuer le signal lorsque le seuil est inférieur.

C'est pour cela qu'il faut conserver l'espace.

Reply:
ok, je vais regarder partout. Merci.